File: docs/Foundation/Blocks/BlockPrinciple/BlockPrinciple/ViewController.m
//
//  ViewController.m
//  BlockPrinciple
//
//  Created by Rake Yang on 2019/12/6.
//  Copyright © 2019年 BinaryParadise. All rights reserved.
//

#import "ViewController.h"
#import "IceBox.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    int a = 10;
    int(^myBlock)(id) = ^(id obj) {
        NSLog(@"%d %@", a, obj);
        return 0;
    };
    
    myBlock([IceBox new]);
    NSLog(@"%@", myBlock);
}

@end

File: Examples/TopicExtractor/Example/TopicExtractor/ViewController.swift
//
//  ViewController.swift
//  TopicExtractor
//
//  Created by rakeyang on 03/08/2020.
//  Copyright (c) 2020 rakeyang. All rights reserved.
//

import Cocoa
import Alamofire
import CocoaLumberjack

class ViewController: NSViewController {

    override func viewDidLoad() {
    super.viewDidLoad()    
    // Do any additional setup after loading the view.
    

    }
    
    @IBAction func loginCMR(_ sender: NSButton) {
        let login = NSWindow.init(contentViewController: LoginViewController.init())
        login.center()
        login.orderFront(nil)
    }
    
    @IBAction func fetchTopic(_ sender: Any) {
        Alamofire.request("/student/acourse/HomeworkCenter/InstantRnd.asp?CourseID=bk134a&CID=05190001").responseData { (response) in
            DDLogDebug("\(response)")
        }
        var mrequest = URLRequest.init(url: URL.init(string: "https://y.neverland.life/api")!) as? NSMutableURLRequest
        mrequest?.httpMethod = "POST"
        mrequest?.httpBody = "fjlaifajeligjsleijgslejg".data(using: String.Encoding.utf8)
        var dataTask = URLSession.shared.dataTask(with: mrequest as! URLRequest)
        dataTask.resume()
    }
    
  override var representedObject: Any? {
    didSet {
    // Update the view, if already loaded.
    }
  }


}

